docs/source/conf.py

```diff
@@ -209,7 +209,7 @@
 # (source start file, target name, title,
 #  author, documentclass [howto, manual, or own class]).
 latex_documents = [
-    (master_doc, "skcosmo.tex", u"scikit-COSMO Documentation", author, "manual"),
+    (master_doc, "skcosmo.tex", "scikit-COSMO Documentation", author, "manual"),
```

Comment:
Why is this change part of this PR?

Reply:
The answer to all of these questions is `black` ^_^. See the second commit in the PR.